added statement of need to paper (#81)
galsci/pysm: paper/paper.md

bibliography: paper.bib
---

# Statement of Need

> The most fruitful region of the radio spectrum to study the Early Universe is the microwave region. At those frequencies peaks the Cosmic Microwave Background (CMB) eletromagnetic radiation coming directly from the beginning of the Universe, just 370 thousand years into the 13.8 billion years age of the Universe.

Reviewer: ```suggestion
The Cosmic Microwave Background (CMB) radiation, emitted just 370 thousand years after the Big Bang, is a pristine probe of the Early Universe. After being emitted at hight temperatures, the CMB was redshifted by the subsequent 13.8 billion years of cosmic expansion, such that it is brightest at microwave frequencies today.
```

> However, our own Milky Way galaxy also emits in the microwaves, for example through thermal radiation by interstellar dust grains or through synchrotron emission by relativistic electrons spiralling in magnetic fields.

Reviewer: ```suggestion
However, our own Milky Way galaxy also emits in the microwave, obscuring our view of the CMB.  Examples of this emission are thermal radiation by interstellar dust grains, and synchrotron emission by relativistic electrons spiralling in magnetic fields.
```

Author: should it be "in the microwaves" or in the microwave region?

Reviewer: I would usually say "in the microwave", but perhaps that is a bit colloquial. "In the microwave portion of the spectrum" would be less colloquial. I would avoid "in the microwave region" as it's not clear we're referring to a region of a spectrum, and I don't think "in the microwaves" is correct.

Author: "In the microwave portion of the spectrum"

> Cosmologists need to create syntetic maps of the CMB and of the galactic emission based on available data and on models that properly scale them to different frequencies. Those maps are useful to test data reduction algorithms, to understand residual systematics, to forecast maps produced by future instruments, to run Montecarlo analysis for noise estimation and more.

Reviewer: ```suggestion
Cosmologists need to create synthetic maps of the CMB and of the galactic emission based on available data and on physical models that extrapolate observations to different frequencies. The resulting maps are useful to test data reduction algorithms, to understand residual systematics, to forecast maps produced by future instruments, to run Monte Carlo analysis for noise estimation, and more.
```
